stories/Modal.jsx

```diff
@@ -44,6 +45,13 @@ import {
 */
 import Sidebar from '../src/components/Sidebar';
 
+const StyledDiv = styled.div`
```

Comment:
Adding a 30px padding is better IMO.

Reply:
Instead of `padding: 10%;` ?
I have a general feeling that fixing values as `px` is a bad idea, since it's anti-responsive by definition...
This is something I wanted to ask you.
Shouldn't be using only `rem` or `%` instead of `px` ?

Reply:
You're totally right, I missed that. That's said, I really think we should add some space here. We could gain some clarity.

So go for the `rem`! :)

Reply:
I'll do that, but will also open an issue for a full `px` to `rem` migration.

Reply:
Thanks!